blogs/scripts/scripts.js

```diff
@@ -571,6 +571,133 @@ export async function addBackOfficeMetaTags() {
   }
 }
 
+/**
+ * loads a block named 'header' into header
+ */
+
+async function loadKeysightHeader(header) {
+  const jqueryScript = document.createElement('script');
+  jqueryScript.type = 'text/javascript';
+  jqueryScript.innerHTML = '';
+  jqueryScript.src = 'https://ajax.googleapis.com/ajax/libs/jquery/3.5.0/jquery.min.js';
```

Comment:
I'd use the existing `loadScript` function for this instead. I'd also recommend that you put jQuery into the project so it can be loaded from the same domain, rather than a 3rd party domain. There is a non-trivial cost of loading resources from external domains.